firefox-ios/Client/Frontend/Autofill/Address/Edit/AddressModifiedStatus.swift

```diff
@@ -36,8 +36,7 @@ enum AddressModifiedStatus {
 
     var message: String {
         switch self {
-        case .saved: return .Addresses.Settings.Edit.AddressSavedConfirmation
-        case .updated: return .Addresses.Settings.Edit.AddressUpdatedConfirmation
+        case .saved, .updated: return .Addresses.Settings.Edit.AddressSavedConfirmation
```

Comment:
@issammani even though they are same English strings, lets add a new string for this but change the comment to reflect what this means. Its mainly for translations other than English

Reply:
Cc @razvanlitianu

Reply:
Ok updated in https://github.com/mozilla-mobile/firefox-ios/pull/21857/commits/ce49c7d347403121c219f8bd920b0d6bcbeea34a. I have a question regarding the version though ? I set it to `v130` with the intention that we can backport this. If that's not possible with strings then I can add a `struct v131` and update versions accordingly.

Reply:
I think we missed string freeze for 131. @nbhasin2 should this be 132? or even 133, we usually add the version we are targeting for release.

Reply:
Yes. exactly. @nbhasin2 told me yesterday 😢 So what we discussed is keeping things as is and adding a V2 string for now. And since we only need english string translations shouldn't be a problem. I made the change in https://github.com/mozilla-mobile/firefox-ios/pull/21857/commits/8db40fc4019ecc0ba233224f857bd8aac437d587